upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -13,6 +13,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+/******************************************************************************
```

Comment:
Fixed in https://github.com/kubernetes/kops/pull/694/commits/d7c209fdbc2ce9bad2087984fa4edd277a68cf9b